src/components/calcite-accordion/readme.md

```diff
@@ -1,16 +1,6 @@
 # calcite-accordion
 
-Calcite-accordion can be used to present content in collapseable sections. Configuration options exist for visual style (theme, icon-position, scale, appearance), and selection functionality. Selection modes include "multi", "single", and "single-persist".
```

Comment:
Is there a reason you dropped the description when you moved this over?

Reply:
I updated part of it, but it wasn't dropped.

Reply:
nvm, read the diff wrong